### PR TITLE
feature/email-event-filter

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
## PR Overview
**This PR will address the following Issue/Feature:** FR #96 

**This PR will result in the following new package version:** `v0.9.1`

This is simply a documentation and under the hood update based on the updates within [PR #104](https://github.com/fivetran/dbt_hubspot_source/pull/104) from the source package. As such, no breaking updates will be reflected.

**Please detail what change(s) this PR introduces and any additional information that should be known during the review of this PR:**

This PR mainly includes documentation around the newly added `hubspot_using_filtered_events` variable (true by default) that changes the behavior of the `stg_hubspot__email_event` staging model to filter out filtered events if the variable is set to `false`. Please see [PR #104](https://github.com/fivetran/dbt_hubspot_source/pull/104) for more details around the upstream changes.

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [X] dbt compile
- [X] dbt run –full-refresh
- [X] dbt run
- [X] dbt test
- [X] dbt run –vars '{hubspot_using_filtered_events: false}'

Before marking this PR as “ready for review” the following have been applied:
- [X] The appropriate issue has been linked and tagged
- [X] You are assigned to the corresponding issue and this PR
- [ ] BuildKite integration tests are passing

### Detailed Validation
Please acknowledge that the following validation checks have been performed prior to marking this PR as “ready for review”:
- [X] You have validated these changes and assure this PR will address the respective Issue/Feature.
- [X] You are reasonably confident these changes will not impact any other components of this package or any dependent packages.
- [X] You have provided details below around the validation steps performed to gain confidence in these changes.

Please see [PR #104](https://github.com/fivetran/dbt_hubspot_source/pull/104) for the validation steps taken to ensure the changes are behaving as expected in the source. With comfort around these changes, I feel the downstream implications will behave the same and as intended.

### Standard Updates
Please acknowledge that your PR contains the following standard updates:
- Package versioning has been appropriately indexed in the following locations:
    - [X] indexed within dbt_project.yml
    - [X] indexed within integration_tests/dbt_project.yml
- [X] CHANGELOG has individual entries for each respective change in this PR
    
- [X] README updates have been applied (if applicable)
    
- [N/A] DECISIONLOG updates have been updated (if applicable)
- [N/A] Appropriate yml documentation has been added (if applicable)

### dbt Docs
Please acknowledge that after the above were all completed the below were applied to your branch:
- [X] docs were regenerated (unless this PR does not include any code or yml updates)

### If you had to summarize this PR in an emoji, which would it be?
 
📧

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+# dbt_hubspot v0.9.1
+## Feature Updates
+- A new variable was added `hubspot_using_filtered_events` to allow package users to remove filtered email events from the `stg_hubspot__email_event` staging model as well as the relevant downstream reporting models. This is crucial for HubSpot users who greatly take advantage of marking events as filtered in order to provide accurate reporting. 
+  - The `hubspot_using_filtered_events` variable is `true` by default. Set the variable to `false` to filter out specified email events in your staging and downstream models.
+
+## Under the Hood
+- The `email_event_data.csv` seed file was updated to include events that are listed as `true` for filtered_events. This is to effectively test the above mentioned feature update. 
+- Included `hubspot_using_filtered_events: false` as a variable declared in the final `run_models.sh` step to ensure our integration tests gain coverage over this new feature and variable.
```
Comment (fivetran-reneeli) [inline CHANGELOG.md:8]: I'm not seeing this config in the run_models.sh script?

File: .buildkite/scripts/run_models.sh [hunk 2]
```diff
@@ -21,6 +21,5 @@ dbt run --target "$db" --full-refresh
 dbt test --target "$db"
 dbt run --vars '{hubspot_marketing_enabled: true, hubspot_sales_enabled: false}' --target "$db" --full-refresh
 dbt run --vars '{hubspot_marketing_enabled: true, hubspot_contact_merge_audit_enabled: true, hubspot_sales_enabled: false}' --target "$db" --full-refresh
-dbt run --vars '{hubspot_marketing_enabled: false, hubspot_sales_enabled: true}' --target "$db" --full-refresh
-dbt run --vars '{hubspot_marketing_enabled: false, hubspot_sales_enabled: false}' --target "$db" --full-refresh
+dbt run --vars '{hubspot_marketing_enabled: false, hubspot_sales_enabled: true, hubspot__pass_through_all_columns: tru}' --target "$db" --full-refresh
```
Comment (fivetran-reneeli) [inline .buildkite/scripts/run_models.sh:24]: typo 

```suggestion
dbt run --vars '{hubspot_marketing_enabled: false, hubspot_sales_enabled: true, hubspot__pass_through_all_columns: true}' --target "$db" --full-refresh
```

**Review by fivetran-reneeli — CHANGES_REQUESTED** [review]
Looks good, just have very minor remarks!

File: CHANGELOG.md (hunk 1 above)
Comment (fivetran-joemarkiewicz) [reply CHANGELOG.md:8]: @fivetran-reneeli thanks for catching this! I must have accidentally removed this when I trimmed one of the run commands. Just added it back in.

**Review by fivetran-reneeli — APPROVED** [review]
Looks good! Reminder to update package deps

**Pull request merged**